### PR TITLE
Beam 2.55.0 website updates

### PR DESCRIPTION
**Pull request by Abacn**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+* Fixed SpannerIO.readChangeStream to support propagating credentials from pipeline options
+  to the getDialect calls for authenticating with Spanner (Java) ([#30361](https://github.com/apache/beam/pull/30361)).
+
+## Security Fixes
+
+* Go SDK base container image moved to distroless/base-nossl-debian12, reducing vulnerable container surface to kernel and glibc ([#30011](https://github.com/apache/beam/pull/30011)).
+
+## Known Issues
+
+*N/A
```
Comment (tvalentyn) [inline website/www/site/content/en/blog/beam-2.55.0.md:78]: FYI there was a recent change that seems to be not reflected here: https://github.com/apache/beam/pull/30701/

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+We are happy to present the new 2.55.0 release of Beam.
+This release includes both improvements and new functionality.
+See the [download page](/get-started/downloads/#2550-2023-03-1{{ TODO }}) for this release.
+
+<!--more-->
+
+For more information on changes in 2.55.0, check out the [detailed release notes](https://github.com/apache/beam/milestone/19).
+
+# [2.55.0] - Unreleased
```
Comment (tvalentyn) [inline website/www/site/content/en/blog/beam-2.55.0.md:30]: todo: date.

File: website/www/site/content/en/blog/beam-2.55.0.md (hunk 1 above)
Comment (Abacn) [reply website/www/site/content/en/blog/beam-2.55.0.md:78]: thanks, will sync with master's README once

**Comment by Abacn** [comment]
R: @tvalentyn updated, PTAL

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+* The Python SDK will now include automatically generated wrappers for external Java transforms! ([#29834](https://github.com/apache/beam/pull/29834))
+
+## I/Os
+
+* Added support for handling bad records to BigQueryIO ([#30081](https://github.com/apache/beam/pull/30081)).
+  * Full Support for Storage Read and Write APIs
+  * Partial Support for File Loads (Failures writing to files supported, failures loading files to BQ unsupported)
+  * No Support for Extract or Streaming Inserts
+* Added support for handling bad records to PubSubIO ([#30372](https://github.com/apache/beam/pull/30372)).
+  * Support is not available for handling schema mismatches, and enabling error handling for writing to pubsub topics with schemas is not recommended
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:41]:
```suggestion
  * Support is not available for handling schema mismatches, and enabling error handling for writing to Pub/Sub topics with schemas is not recommended
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+* `--enableBundling` pipeline option for BigQueryIO DIRECT_READ is replaced by `--enableStorageReadApiV2`. Both were considered experimental and may subject to change (Java) ([#26354](https://github.com/apache/beam/issues/26354)).
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:42]:
```suggestion
* `--enableBundling` pipeline option for BigQueryIO DIRECT_READ is replaced by `--enableStorageReadApiV2`. Both were considered experimental and subject to change (Java) ([#26354](https://github.com/apache/beam/issues/26354)).
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+## New Features / Improvements
+
+* Allow writing clustered and not time partitioned BigQuery tables (Java) ([#30094](https://github.com/apache/beam/pull/30094)).
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:46]:
```suggestion
* Allow writing clustered and not time-partitioned BigQuery tables (Java) ([#30094](https://github.com/apache/beam/pull/30094)).
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+* Redis cache support added to RequestResponseIO and Enrichment transform (Python) ([#30307](https://github.com/apache/beam/pull/30307))
+* Merged sdks/java/fn-execution and runners/core-construction-java into the main SDK. These artifacts were never meant for users, but noting
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:48]:
```suggestion
* Merged `sdks/java/fn-execution` and `runners/core-construction-java` into the main SDK. These artifacts were never meant for users, but noting
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+* Added Vertex AI Feature Store handler for Enrichment transform (Python) ([#30388](https://github.com/apache/beam/pull/30388))
+
+## Breaking Changes
+
+* Arrow version was bumped to 15.0.0 from 5.0.0 ([#30181](https://github.com/apache/beam/pull/30181)).
+* Go SDK users who build custom worker containers may run into issues with the move to distroless containers as a base (see Security Fixes).
+  * The issue stems from distroless containers lacking additional tools, which current custom container processes may rely on.
+  * See https://beam.apache.org/documentation/runtime/environments/#from-scratch-go for instructions on building and using a custom container.
+* Python SDK has changed the default value for the `--max_cache_memory_usage_mb` pipeline option from 100 to 0. This option was first introduced in 2.52.0 SDK. This change restores the behavior of 2.51.0 SDK, which does not use the state cache. If your pipeline uses iterable side inputs views, consider increasing the cache size by setting the option manually. ([#30360](https://github.com/apache/beam/issues/30360)).
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:58]:
```suggestion
* Python SDK has changed the default value for the `--max_cache_memory_usage_mb` pipeline option from 100 to 0. This option was first introduced in the 2.52.0 SDK version. This change restores the behavior of the 2.51.0 SDK, which does not use the state cache. If your pipeline uses iterable side inputs views, consider increasing the cache size by setting the option manually. ([#30360](https://github.com/apache/beam/issues/30360)).
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+
+## Deprecations
+
+* N/A
+
+## Bugfixes
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:64]:
```suggestion
## Bug fixes
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+
+* Fixed SpannerIO.readChangeStream to support propagating credentials from pipeline options
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:66]:
```suggestion
* Fixed `SpannerIO.readChangeStream` to support propagating credentials from pipeline options
```

File: website/www/site/content/en/blog/beam-2.55.0.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+  to the getDialect calls for authenticating with Spanner (Java) ([#30361](https://github.com/apache/beam/pull/30361)).
```
Comment (rszper) [inline website/www/site/content/en/blog/beam-2.55.0.md:67]:
```suggestion
  to the `getDialect` calls for authenticating with Spanner (Java) ([#30361](https://github.com/apache/beam/pull/30361)).
```

**Review by rszper — APPROVED** [review]

**Comment by Abacn** [comment]
merge for now as 2.55.0 download link is now available

**Pull request merged**